pccommon/pccommon/tracing.py

```diff
@@ -249,3 +257,49 @@ def _iter_cql(cql: dict, property_name: str) -> Optional[Union[str, List[str]]]:
                             return result
     # No collection was found
     return None
+
+
+def add_stac_attributes_from_search(search_json: str, request: fastapi.Request) -> None:
+    """
+    Try to add the Collection ID and Item ID from a search to the current span.
+    """
+    collection_id, item_id = parse_collection_from_search(
+        json.loads(search_json), request.method, request.query_params
+    )
+    span = trace.get_current_span()
```

Comment:
`trace` is coming from a new dependency here, and now we're mixing opencensus with opentelemetry. I think we could actually drop the new dependency and instead just get the current span from `request.state.parent_span`. You can see that it is set up in the `trace_request` method when it sets up the additional tracing dimensions.